salt/states/etcd_mod.py

```diff
@@ -0,0 +1,264 @@
+# -*- coding: utf-8 -*-
+
+'''
+Manage etcd Keys
+================
+
+.. versionadded:: Beryllium
+
+:depends:  - python-etcd
+
+This state module supports setting and removing keys from etcd.
+
+Salt Master Configuration
+-------------------------
+
+To work with an etcd server you must configure an etcd profile in the Salt
+Master configuration, for example:
+
+.. code-block:: yaml
+    my_etd_config:
+      etcd.host: 127.0.0.1
+      etcd.port: 4001
+
+You can also configure etcd without a profile however it is recommended that
+you use profiles:
+
+.. code-block:: yaml
+    etcd.host: 127.0.0.1
+    etcd.port: 4001
+
+Available Functions
+-------------------
+
+- ``set``
+
+  This will set a value to a key in etcd. Changes will be returned if the key
+  has been created or the value of the key has been updated. This
+  means you can watch these states for changes.
+
+  .. code-block:: yaml
+      /foo/bar/baz:
+        etcd.set:
+          - value: foo
+          - profile: my_etcd_config
+
+- ``wait_set``
+
+  Performs the same functionality as``set`` but only if a watch requisite is ``True``.
+
+  .. code-block:: yaml
+
+      /some/file.txt:
+        file.managed:
+          - source: salt://file.txt
+
+      /foo/bar/baz:
+        etcd.wait_set:
+          - value: foo
+          - profile: my_etcd_config
+          - watch:
+            - file: /some/file.txt
+
+- ``rm``
+
+  This will delete a key from etcd. If the key exists then changes will be
+  returned and thus you can watch for changes on the state, if the key does
+  not exist then no changes will occur.
+
+  .. code-block:: yaml
+      /foo/bar/baz:
+        etcd.rm:
+          - profile: my_etcd_config
+
+- ``wait_rm``
+
+  Performs the same functionality as``rm`` but only if a watch requisite is ``True``.
+
+  .. code-block:: yaml
+
+      /some/file.txt:
+        file.managed:
+          - source: salt://file.txt
+
+      /foo/bar/baz:
+        etcd.wait_rm:
+          - profile: my_etcd_config
+          - watch:
+            - file: /some/file.txt
+'''
+
```

Comment:
@krak3n Sorry for the confusion here, but to fix the python3 pylint error, all you need to do is add the following, right here:
```
# Import Python Libs
from __future__ import absolute_import
```